bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_GluGluH_01J_HToAATo4B_M-MASS/SUSY_GluGluH_01J_HToAATo4B_M-MASS_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model NMSSMHET --modelname
+
+generate p p  > h01 $$ w+ w- z , (h01 > a01 a01, a01 > b b~, a01 > b b~)
+generate p p  > h01 j $$ w+ w- z , (h01 > a01 a01, a01 > b b~, a01 > b b~)
```

Comment:
You should write "add process" instead of "generate" in the second line if you want "additional" diagrams

Reply:
Fixed, thanks!